docs/howto/custom-file-storage.txt

```diff
@@ -76,7 +76,13 @@ objects. These are:
 Called by ``Storage.open()``, this is the actual mechanism the storage class
 uses to open the file. This must return a ``File`` object, though in most cases,
 you'll want to return some subclass here that implements logic specific to the
-backend storage system.
+backend storage system. This will raise a ``FileNotFoundError`` exception if the
+file does not exist. To catch: ::
+
+    try:
+        f = myStorage.open()
+    except FileNotFoundError as e:
+        print(f"File {path} not found!", file=sys.stderr)
```

Comment:
This ticket is about an exception that should be raised by `_open()`, not how to catch exceptions from `open()`. Maybe:
```suggestion
backend storage system. The :exc:`FileNotFoundError` exception should be raised
when a file doesn't exist.
```

Reply:
Thank you for your feedback.  I misunderstood the original ask.  I have reflected your feedback in the most recent commit.